webapp/src/ts/modules/error/error.component.ts

```diff
@@ -9,18 +13,55 @@ const errors = {
   404: {
     title: 'error.404.title',
     description: 'error.404.description'
-  }
+  },
+  503: {
+    title: 'error.503.description',
+    description: 'error.503.description'
+  },
 };
+const timeoutDuration = 5000; // 5 seconds
 
 @Component({
   templateUrl: './error.component.html'
 })
-export class ErrorComponent implements OnInit {
+export class ErrorComponent implements OnInit, OnDestroy {
   error;
+  translationsLoaded = false;
+  timeoutElapsed = false;
+  subscriptions: Subscription = new Subscription();
 
-  constructor(private route: ActivatedRoute) {}
+  constructor(
+    private route: ActivatedRoute,
+    private translateService: TranslateService,
+    private store: Store,
+  ) {}
 
-  ngOnInit() {
+  ngOnInit(): void {
     this.error = errors[this.route.snapshot.params.code] || errors['404'];
+    this.translationsLoaded = this.translateService.instant('error.403.description') !== 'error.403.description';
+
+    const subscription = this.store
+      .select(Selectors.getTranslationsLoaded)
+      .subscribe(isLoaded => {
+        if (isLoaded) {
+          this.timeoutElapsed = false;
+        } else {
+          this.startTimeout();
```

Comment:
@njogz I don't fully understand. Why we are giving extra time for setting `this.timeoutElapsed = true`? Maybe a more descriptive function name can help and/or comment.

Reply:
If you add a commit for this, can you please add a space between third-party imports and Medic's imports, that's something we're trying to keep :)

Reply:
I think `this.startTimeout();` should happen in line 43, before the subscription, so it shows the default message if the translations don't resolve after a long time. Because I don't see in the code that `getTranslationsLoaded` will resolve `false`.
You can explain that in the comment in the `startTimeout(){` definition.